mvc/views/view-components.rst

```diff
@@ -21,54 +30,98 @@ New to ASP.NET MVC 6, view components are similar to partial views, but they are
+
+A view component consists of two parts, the class (typically derived from  ``ViewComponent``) and the Razor view which calls methods in the view component class). Like controllers, a view component can be a POCO, but most developers will want to take advantage of the methods and properties available by deriving from ``ViewComponent``.
 
-You can also create a view component that gets and renders data depending on the user's claims. You can add this view component view to the layout page and have it get and render user-specific data throughout the whole application. View components don’t use model binding, and only depend on the data you provide when calling into it. 
+See `ViewComponent properties <https://docs.asp.net/projects/api/en/latest/autoapi/Microsoft/AspNet/Mvc/ViewComponent/index.html#properties>`__ and `ViewComponent Class <https://docs.asp.net/projects/api/en/latest/autoapi/Microsoft/AspNet/Mvc/ViewComponent/index.html>`__.
 
-A view component consists of two parts, the class (typically derived from  ``ViewComponent``) and the Razor view which calls methods in the view component class. Like controllers, a view component can be a POCO, but most users will want to take advantage of the methods and properties available by deriving from ``ViewComponent``.
+Creating a view component
+---------------------------
+
+This section contains the high level requirements to create a view component. Later in the article we'll examine each step in detail and create a view component.
+
+The view component class
+^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 A view component class can be created by any of the following:
 
-- Deriving from `ViewComponent`.
-- Decorating the class with the ``[ViewComponent]`` attribute, or deriving from a class with the ``[ViewComponent]`` attribute.
-- Creating a class where the name ends with the suffix *ViewComponent*.
+- Deriving from `ViewComponent`
+- Decorating a class with the ``[ViewComponent]`` attribute, or deriving from a class with the ``[ViewComponent]`` attribute
+- Creating a class where the name ends with the suffix *ViewComponent*
 
-Like controllers, view components must be public, non-nested, non-abstract classes.
+Like controllers, view components must be public, non-nested, and non-abstract classes.
 
-Examine the ViewComponent class
---------------------------------
+View component methods
+^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-- Examine the *src\\TodoList\\ViewComponents\\PriorityListViewComponent.cs* file:
+- Contain ``Invoke`` or ``InvokeAsync`` methods that return an ``IViewComponentResult``
+- Fully support constructor `Dependency Injection <https://docs.asp.net/en/latest/fundamentals/dependency-injection.html>`__
+- Typically initialize a model and pass it to a view by calling the ``ViewComponent`` ``View`` method. 
+- Parameters come from the calling method, not HTTP, there is no model binding. This allows you to overload ``Invoke``/``InvokeAsync``  (which you can't do with controllers).
```

Comment:
We removed this from RC2. Not sure what you're targeting with this article. Here's the relevant announcement: https://github.com/aspnet/Announcements/issues/145